### PR TITLE
Task 7 ssr

### PR DESCRIPTION
**Pull request by sixtyXi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,12 +59,16 @@
...
-    "webpack-merge": "^4.2.1"
+    "webpack-dev-middleware": "^3.6.2",
+    "webpack-hot-middleware": "^2.24.4",
+    "webpack-hot-server-middleware": "^0.6.0",
+    "webpack-merge": "^4.2.1",
+    "webpack-node-externals": "^1.7.2"
   },
   "dependencies": {
     "classnames": "^2.2.6",
-    "express": "^4.16.4",
```
Comment (IrinaShilina) [inline package.json]: Why did you move Express to dev dependencies?

Comment (IrinaShilina) [reply package.json]: Fix for sorting:
```
    switch (sortBy) {
      case SORT_FILTERS[0].value:
        return movies.sort(
          (firstMovie, secondMovie) =>
            new Date(secondMovie[SORT_FILTERS[0].value]) -
            new Date(firstMovie[SORT_FILTERS[0].value])
        );
      case SORT_FILTERS[1].value:
        return movies.sort(
          (firstMovie, secondMovie) =>
            secondMovie[SORT_FILTERS[1].value] - firstMovie[SORT_FILTERS[1].value]
        );

      default:
        return movies;
    }
```

Comment (sixtyXi) [reply package.json]: Fixed.
